packages/mui-icons-material/package.json

```diff
@@ -83,5 +83,21 @@
   },
   "engines": {
     "node": ">=14.0.0"
+  },
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "node": "./index.js",
+      "import": "./esm/index.js",
+      "require": "./index.js"
+    },
+    "./*": {
+      "types": "./*.d.ts",
+      "node": "./*.js",
+      "import": "./esm/*.js",
+      "require": "./*.js"
+    },
+    "./esm/*": "./esm/*.js",
```

Comment:
Why has this been added? For backward compatibility ?

Reply:
Exactly, someone might have added
```tsx
  resolve: {
    alias: [
      {
        find: /^@mui\/icons-material\/(.*)/,
        replacement: '@mui/icons-material/esm/$1',
      },
    ],
  },
```
to their vite config. It means under the hood their bundler is trying to resolve `@mui/icons-material/esm/Add` from the icons package. If we don't add an export for this, that module request will follow the `./*` export and try to access `@mui/icons-material/esm/esm/Add.js`